### ISSUE TITLE
Many trailing spaces are added after : even with to_empty=True

### ISSUE THREAD

**Issue by Ark-kun** [abarker/strip-hints#10]
When the function has multiline return type annotation, the stripped version has multiple spaces after the `:` symbol (the original function has none).
```
strip_string_to_string(to_empty=True, code_string='''
def foo(
  param1: int,
  param2: str = "zzz",
) -> NamedTuple('Outputs', [
  ('out1', str),
  ('out2', int),
]):
  pass
''')
```
```
'\ndef foo(\n  param1 ,\n  param2  = "zzz",\n):   \n   \n   \n\n  pass\n'`
```

**Comment by abarker** [comment]
It's supposed to do that in order to preserve line numbers in error messages.  Using `to_empty` no  longer guarantees that the column numbers are preserved, but line numbers are still supposed to be.  The program purposely excludes stripping NL tokens (non-logical newlines) to achieve this.

Does the extra space cause problems?  The code is still syntactically valid.

I added an option `strip_nl` to strip NL tokens, too, but using that option no longer preserves line numbers.  I pushed it to GitHub but it isn't yet on PyPI and isn't documented.  I also fixed a limitation of annotated assignment stripping when they contained NL characters inside type annotations which were then stripped.  The `no_equal_move` option was added to disable that moving of the assignment line to fix the problem.
